kyuubi-server/src/main/scala/org/apache/kyuubi/client/KyuubiSyncThriftClient.scala

```diff
@@ -30,7 +30,67 @@ import org.apache.kyuubi.operation.FetchOrientation.FetchOrientation
 import org.apache.kyuubi.session.SessionHandle
 import org.apache.kyuubi.util.ThriftUtils
 
-class KyuubiSyncThriftClient(protocol: TProtocol) extends TCLIService.Client(protocol) {
+trait KyuubiSyncThriftClient {
```

Comment:
Is this necessary to create this？

Reply:
Removed dynamic proxy and send delegation tokens only during `ExecuteStatement#waitStatementComplete`.
No longer need to change `KyuubiSyncThriftClient` to a trait.